src/smt/set_defaults.cpp

```diff
@@ -627,12 +635,12 @@ void SetDefaults::setDefaults(LogicInfo& logic, Options& opts)
     opts.smt.produceModels = true;
   }
 
-  /////////////////////////////////////////////////////////////////////////////
-  // Widen logic
-  /////////////////////////////////////////////////////////////////////////////
+  // widen the logic
   widenLogic(logic, opts);
-  /////////////////////////////////////////////////////////////////////////////
+}
```

Comment:
Suggestion: move the `widenLogic` function to the top level?

Reply:
I think this will eventually be inlined into finalizeLogic.  I'm leaving it here for now.